drivers/SmartThings/zwave-garage-door-opener/profiles/eco-zw-gdo-profile.yml

```diff
@@ -0,0 +1,81 @@
+name: eco-zw-gdo-profile
+components:
+  - id: main
+    capabilities:
+    - id: doorControl
+      version: 1
+    - id: temperatureMeasurement
+      version: 1
+    - id: firmwareUpdate
```

Comment:
I doubt this device is able to be FW updated via SmartThings. AFIAK this feature for z-wave is not used in production and device FW files are manually added by SmartThings to enable it for particular devices.

Reply:
Got it.